build/Targets/Settings.targets

```diff
@@ -150,7 +150,8 @@
 
         <Otherwise>
           <PropertyGroup>
-            <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25604-Preview4</VisualStudioBuildToolsNuGetPackagePath>
+            <!-- TODO: Detect Preview 4/5 versus RC -->
+            <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25825-RC</VisualStudioBuildToolsNuGetPackagePath>
```

Comment:
I feel like it is completely reasonable to restrict ourselves to RC or later (and then RTM or later once that is out).